tests/integration/test_api.py

```diff
@@ -185,6 +185,16 @@ def test_annotate_insight_authenticated(client):
     assert insight.items() > {"username": "a", "annotation": -1, "n_votes": 0}.items()
     assert "completed_at" in insight
 
+    # to test if "annotated_result" is being saved in the database
+    insight = next(
+        ProductInsight.select()
+        .where(ProductInsight.annotated_result.is_null(False))
+        .dicts()
+        .iterator()
+    )
+    assert insight.items() > {"username": "a", "annotation": -1, "n_votes": 0}.items()
+    assert "completed_at" in insight
```

Comment:
You already have the insght, the only thing you have to test is that annotated_result is the expected value !

```suggestion
    assert insight.completed_at == "updated"
```